### DISCUSSION TITLE
My blockquote render hook for alert boxes (callouts)

### DISCUSSION THREAD

**Discussion by huanlin** [google/docsy#2080] [Show and tell]
Hugo supports [blockquote render hooks](https://gohugo.io/render-hooks/blockquotes/) since v0.132.0, which makes it easy to create beautiful alert boxes with the simple blockquote syntax. This is great because I don't need to use shortcodes for creating alert boxes anymore.

So here are examples and the implementation applied to my Hugo+Docsy website: [Hugo Blockquote Render Hook](https://huanlin.cc/blog/2024/09/09/hugo-blockquote-render-hook/)

Preview:

![image](https://github.com/user-attachments/assets/2658f058-77bb-45c6-8716-a25cf0c1184b)

Cheers.

**Comment by deining** [comment]
Thnaks for your input, sounds interesting:

> So here are examples and the implementation applied to my Hugo+Docsy website: [Hugo Blockquote Render Hook](http://localhost:1856/blog/2024/09/09/hugo-blockquote-render-hook/)

You are linking to your local PC, no one will be able to see that (except for you 😁). Can you provide a link that is publicly available?

**Reply by huanlin** [reply]
Hi @deining 

Thank you for letting me know that I I've pasted a wrong URL.
I've fixed the URL, or click here: [My Hugo Blockquote Render Hook](https://huanlin.cc/blog/2024/09/09/hugo-blockquote-render-hook/)

**Comment by fekete-robert** [comment]
Looks interesting, thanks for sharing!
